.github/workflows/test-api-contract.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - add-api-docs-workflow
```

Comment:
if/once we happy to merge i will make a final commit to this PR to remove this branch - just needed as part of testing this PR.